presto-main/src/main/java/com/facebook/presto/server/security/AuthenticationFilter.java

```diff
@@ -176,14 +176,22 @@ private static void skipRequestBody(HttpServletRequest request)
         }
     }
 
-    private static class CustomHttpServletRequestWrapper
+    public static CustomHttpServletRequestWrapper wrapWithHeader(HttpServletRequest request, Principal principal)
```

Comment:
Name the method `withPrincipalAndCustomHeaders`

Reply:
Okay